protostar/commands/test/test_suite.py

```diff
@@ -5,6 +5,7 @@
 
 @dataclass(frozen=True)
 class TestCase:
+    test_path: Path
     test_fn_name: str
     setup_fn_name: Optional[str] = None
```

Comment:
Maybe `TestCase` should have a reference to test suite instead of `test_path`.

Reply:
Didn't think about this :+1: Done.